docs/source/index.md

```diff
@@ -20,113 +20,183 @@ ExecuTorch provides support for:
 
 * **Strong Model Support** LLMs (Large Language Models),
   CV (Computer Vision), ASR (Automatic Speech Recognition), TTS (Text To Speech)
-* **All Major Platforms** Android, Mac, Linux, Windows
+* **All Major Platforms** Android, iOS, macOS, Linux, Windows, Embedded
 * **Rich Acceleration Support** Apple, Arm, Cadence, MediaTek, NXP, OpenVino, Qualcomm, Vulkan, XNNPACK
 
 ### Documentation Navigation
-#### Introduction
+
+#### Getting Started
 - [Overview](intro-overview)
 - [How it Works](intro-how-it-works)
-- [Getting Started with Architecture](getting-started-architecture)
-- [Concepts](concepts)
-#### Usage
-- [Getting Started](getting-started)
-- [Using Executorch Export](using-executorch-export)
-- [Using Executorch on Android](using-executorch-android)
-- [Using Executorch on iOS](using-executorch-ios)
-- [Using Executorch with C++](using-executorch-cpp)
-- [Runtime Integration](using-executorch-runtime-integration)
-- [Troubleshooting](using-executorch-troubleshooting)
+- [Architecture & Components](getting-started-architecture)
+- [Core Concepts](concepts)
 - [Building from Source](using-executorch-building-from-source)
```

Comment:
Can we have a section called 

"Quick Start" that contains only a few sections

- [Getting Started](getting-started)
- [Model Export & Lowering](using-executorch-export)
- [Building from Source](using-executorch-building-from-source)